### DISCUSSION TITLE
Display is lit but nothing displayed

### DISCUSSION THREAD

**Discussion by Sercurio** [ElectroTechnique/electrotechnique.github.io#82] [Hardware Issues]
Hi,

I finished my build of the tsynth, but when in plug in usb the screen stay lit with nothing displayed.

I uploaded the source code and the serial monitor show this:
```
SD card is connected
Ignoring Dir
1:Solina
2:Moog Bass
Set Patch: Moog Bass
Set Patch: Overhead
Set Patch: Phase Bass
Set Patch: Metal FM
Set Patch: Phase Bass
Set Patch: Overhead
Set Patch: Moog Bass
```

We can see that the ENC1 correctly working.

I'm working with this screen: https://fr.aliexpress.com/item/1005001621706688.html?gatewayAdapt=glo2fra&spm=a2g0o.9042311.0.0.532b6c37BPUSpW

Can I do something for debugging further ?

**Comment by Electrotechnique** [comment]
Hello, the display looks the same but the actual module they solder to the PCB varies between sources. There should be some kind of image though. Firstly, check the soldering - the usual culprit. Also check the pins of the Teensy are actually making good contact with the header pins -  another source of problems.
